statistics/handle/lru_cache_test.go

```diff
@@ -23,10 +23,10 @@ import (
 )
 
```

Comment:
Can we add tests for add/drop index/column?

Reply:
updated, covered in TestLRUFreshMemUsage